### PR TITLE
feat: add data augmentation for text notebook

### PR DESCRIPTION
**Pull request by kweinmeister**
Add notebook to demonstrate use of synonym replacement, backtranslation, and noise injection.

### CODE REVIEW

File: gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb [hunk 1]
```diff
@@ -0,0 +1,552 @@
+{
```
Comment (holtskinner) [inline gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb:1]: <p>I'd recommend adding BigQuery or BigFrames into the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (holtskinner) [inline gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb:1]: <p>Does the Vertex AI API also need to be enabled?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (holtskinner) [inline gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    # https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.ml.llm.GeminiTextGenerator
</pre><p>I'd recommend putting this in a markdown cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (holtskinner) [inline gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb:1]: <p>Was this meant to be a markdown link?  "Understanding Back-Translation at Scale"[<a href="https://arxiv.org/pdf/1808.09381.pdf%5D" target="_blank">https://arxiv.org/pdf/1808.09381.pdf]</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Review by holtskinner — APPROVED** [review]
Minor suggestions

File: gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb (hunk 1 above)
Comment (kweinmeister) [reply gemini/use-cases/data-augmentation/data_augmentation_for_text.ipynb:1]: Not that I'm aware of; don't see it in this notebook: https://github.com/googleapis/python-bigquery-dataframes/blob/main/notebooks/getting_started/ml_fundamentals_bq_dataframes.ipynb

**Pull request merged**